script/_wpantund

```diff
@@ -0,0 +1,67 @@
+#!/bin/sh
+#
+#  Copyright (c) 2017, The OpenThread Authors.
+#  All rights reserved.
+#
+#  Redistribution and use in source and binary forms, with or without
+#  modification, are permitted provided that the following conditions are met:
+#  1. Redistributions of source code must retain the above copyright
+#     notice, this list of conditions and the following disclaimer.
+#  2. Redistributions in binary form must reproduce the above copyright
+#     notice, this list of conditions and the following disclaimer in the
+#     documentation and/or other materials provided with the distribution.
+#  3. Neither the name of the copyright holder nor the
+#     names of its contributors may be used to endorse or promote products
+#     derived from this software without specific prior written permission.
+#
+#  THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+#  AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+#  IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+#  ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+#  LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+#  CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+#  SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+#  INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+#  CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+#  ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+#  POSSIBILITY OF SUCH DAMAGE.
+#
+
+wpantund_uninstall()
+{
+    if which systemctl; then
+        sudo systemctl stop wpantund || true
+        sudo systemctl disable wpantund || true
+    fi
+    sudo killall wpantund || true
+
+    test -d $BUILD_DIR/wpantund || return 0
+
+    (cd $BUILD_DIR/wpantund &&
+        (test ! -f config.status || sudo make uninstall))
+}
+
+wpantund_install()
+{
+    test -d $BUILD_DIR/wpantund || (cd $BUILD_DIR &&
+        git clone --depth 1 https://github.com/openthread/wpantund.git)
+
+    (cd $BUILD_DIR/wpantund &&
+        (test -f configure || ./bootstrap.sh) &&
```

Comment:
After running `./setup`, I received the output below. Maybe a missing dep or config issue?:

vagrant@precise64:~/borderrouter/script$ ./setup     
otbr-web: no process found
otbr-agent: no process found
wpantund: no process found
Cloning into 'wpantund'...
remote: Counting objects: 1759, done.
remote: Compressing objects: 100% (1065/1065), done.
remote: Total 1759 (delta 757), reused 1536 (delta 630), pack-reused 0
Receiving objects: 100% (1759/1759), 2.15 MiB | 912 KiB/s, done.
Resolving deltas: 100% (757/757), done.
 ***************************** 
./bootstrap.sh: 33: ./bootstrap.sh: [[: not found

./bootstrap.sh: 36: ./bootstrap.sh: [[: not found
 *** autoreconf failed with error code 1